tutorials/tts/FastPitch_Adapter_Finetuning.ipynb

```diff
@@ -89,7 +97,8 @@
    "source": [
     "# .nemo files for your pre-trained FastPitch and HiFiGAN\n",
     "pretrained_fastpitch_checkpoint = \"\"\n",
-    "finetuned_hifigan_on_multispeaker_checkpoint = \"\""
+    "finetuned_hifigan_on_multispeaker_checkpoint = \"\"\n",
```

Comment:
Are users expected to download these checkpoints outside of this notebook and fill these in by themselves? If so, can you make this explicit?